code/datums/logging/_log_entry.dm

```diff
@@ -0,0 +1,177 @@
+/datum/log_entry
+	/*
+	 * The version of this log entry, based on semver: https://semver.org/
+	 *
+	 * This is used to guarantee to automated tooling that the data it is parsing
+	 * is what it is expecting.
+	 *
+	 * For backwards incompatible changes, such as REMOVING fields, or CHANGING their meaning,
+	 * update the MAJOR version number. For instance, 1.2.3 -> 2.0.0.
+	 *
+	 * For backwards compatible additions, such as ADDING fields,
+	 * update the MINOR version number. For instance, 1.2.3 -> 1.3.0.
+	 *
+	 * Otherwise, if you are changing the datum AT ALL, such as to refactor its code or fix an edge case,
+	 * update the PATCH version number. For instance, 1.2.3 -> 1.2.4.
+	 * This is so that in the event your code breaks the logging, automated tooling can isolate it.ted for.
+	 */
+	VAR_PRIVATE/version = "0.0.0"
```

Comment:
You'll need to unit test that every version matches `[0-9]+\.[0-9]+\.[0-9]+` and isn't 0.0.0